buildbot/osuosl/master/config/builders.py

```diff
@@ -473,7 +473,8 @@
     # problems sooner rather than later.
     {'name' : "clang-aarch64-sve-vla",
     'tags'  : ["clang"],
-    'workernames' : ["linaro-g3-01", "linaro-g3-02", "linaro-g3-03", "linaro-g3-04"],
+    'workernames' : ["linaro-g3-01", "linaro-g3-02", "linaro-g3-03", "linaro-g3-04",
+                     "linaro-g4-01", "linaro-g4-02"],
```

Comment:
I think, it's dangerous to run the same builder on workers with different hardware.  I suggest we add clang-aarch64-sve2-vla and clang-aarch64-sve2-vla-2stage builders.
The intention of running buildbots both on Graviton3 and Graviton4 machines is to catch bugs when SVE vectorization works on one piece of hardware, but fails on another.

Reply:
Ok I will reduce this PR to the workers only and do the builders separately.